test/functional/p2p-timeouts.py

```diff
@@ -65,11 +67,17 @@ def run_test(self):
         no_verack_node.send_message(msg_ping())
         no_version_node.send_message(msg_ping())
 
-        sleep(32)
-
-        assert not no_verack_node.connected
-        assert not no_version_node.connected
-        assert not no_send_node.connected
+        expected_timeout_logs = [
+            "version handshake timeout from 0",
+            "socket no message in first 3 seconds, 1 0 from 1",
+            "socket no message in first 3 seconds, 0 0 from 2",
+        ]
+
+        with self.nodes[0].assert_debug_log(expected_msgs=expected_timeout_logs):
+            sleep(2)
```

Comment:
Should be

```suggestion
            sleep(3)
```

because of https://github.com/dashpay/dash/pull/3450/commits/d23536453400b07ad8fd8d54024b6b1a6ee9b73d